doc/developer-guide/upgrading.rst

```diff
@@ -370,11 +370,11 @@ The following include files have been renamed:
 HTTPS support
 ^^^^^^^^^^^^^
 
-HTTPS support was moved from the core into a new module, :ref:`mod_ssl`.
+HTTPS support was moved from the core into a new module, *mod_ssl*.
```

Comment:
Is this still accurate? I don’t see any mod_ssl. We moved this to the *core*, right?